projects/hslayers/src/common/layman/access-rights/layman-access-rights.html

```diff
@@ -46,7 +46,7 @@
     <div class="hs-save-map-users-list p-1 my-1">
         <div class="d-flex flex-row justify-content-between align-items-start p-1"
             *ngFor="let user of allUsers | filter: userFilter">
-            <span class="w-50">{{user.name}}</span>
+            <span class="w-50">{{getUserName(user)}}</span>
```

Comment:
I don't like function calls inside templates. You never know which one of them becomes slow at some point

Reply:
Sorry about this, made in a hurry and forgot to stay out of functions inside loops.